src/modules/client/mixins/helperMixin.js

```diff
@@ -121,18 +121,20 @@ export const helperMixin = {
         NotifyPositive('Email envoyé');
 
         await this.getUserHelpers();
-        this.openNewHelperModal = false;
+        this.resetAddHelperForm();
```

Comment:
comme pour l'autre PR, ici tu laisse le ` this.openNewHelperModal = false;` et le `this. resetAddHelperForm()` est appelé par le @hide. sinon `resetAddHelperForm` est appelé 2 fois...